PYME/LMVis/Extras/pointSetGeneration.py

```diff
@@ -105,6 +105,37 @@ class ImageSource(PointSource):
     #name = Str('Density Image')
     #foo = Enum([1,2,3,4])
 
+    helpInfo = {
+        'points_per_pixel' : '''
+Select average number of points (dye molecules or docking sites) per pixel in the image mask.
+The number can be a floating point fraction, e.g. 0.1 or 3.7, a suitable choice will depend
```

Comment:
```suggestion
The number is a floating point fraction, e.g. 0.1, and shouldn't exceed 1. It is used for Monte-Carlo rejection of positions and larger values (>~0.2) will result in images which have visible pixel-grid structure because the Monte-Carlo sampling is no longer a good approximation to random sampling over the grid. If this is a problem for your application / you can't get high enough density without a high acceptance fraction, use an up-sampled source image with a smaller pixel size. 
```